packages/webcert/src/feature/certificate/Inputs/UeSickLeavePeriod/DateRangePicker.tsx

```diff
@@ -168,10 +176,13 @@ const DateRangePicker: React.FC<Props> = ({
   const toggleShowValidationError = (fromDate: string | null, toDate: string | null) => {
     const updatedValidationErrors = { ...validations }
 
+    updatedValidationErrors.notCompleteDate = false
+    updatedValidationErrors.invalidDatePeriod = false
+
     if (fromDate && toDate && isBefore(getValidDate(toDate)!, getValidDate(fromDate)!)) {
       updatedValidationErrors.invalidDatePeriod = true
-    } else {
-      updatedValidationErrors.invalidDatePeriod = false
+    } else if (isShowValidationError && ((fromDate && !toDate) || (toDate && !fromDate))) {
```

Comment:
You're only checking `isShowValidationError` for the `notCompleteDate`-validation, is that correct?
Also, I thought that `isShowValidationError` only controls rendering of validation message, not whether validation occurs.